articles/quickstart/backend/aspnet-core-webapi/v2/01-authorization.md

```diff
@@ -43,27 +43,34 @@ Also, update the `appsettings.json` file in your project with the correct **Doma
 
 <%= include('../../_includes/_api_auth_preamble') %>
 
```

Comment:
I think we need to make a note here somewhere that ASP.NET Core has middleware which handles the validation of the token for the user, so they do not have to go through the process of manually validating the token themselves